src/igvfd/schemas/changelogs/construct_library_set.md

```diff
@@ -1,5 +1,9 @@
 ## Changelog for *`construct_library_set.json`*
 
+### Minor changes since schema version 4
+
+* Add linking to reference files for `large_scale_gene_list` and `large_scale_loci_list`.
```

Comment:
Add ReferenceFile linkTo to `large_scale_gene_list` and `large_scale_loci_list`.

Reply:
changed